src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC, useContext } from 'react';
+import cn from 'classnames';
+
+import { TodoContext } from '../../context/ContextTodo';
+
+import { Todo } from '../../types/Todo';
+
+type TTodoItemProps = {
+  todo: Todo;
+};
+
+export const TodoItem: FC<TTodoItemProps> = ({ todo }) => {
+  const { id, title, completed } = todo;
+
+  const {
+    setIsModalOpen,
+    setSelectedTodo,
+    selectedTodo,
+  } = useContext(TodoContext);
+
+  const handleSelectTodo = (chosenTodo: Todo) => {
+    setSelectedTodo(chosenTodo);
+    setIsModalOpen(true);
+  };
+
+  return (
+    <tr
+      data-cy="todo"
+      className={cn({
+        'has-background-info-light': selectedTodo?.id === id,
```

Comment:
DRY. Create variable for this value `selectedTodo?.id === id` and reuse it here and in other places